generators/hub/templates/static/index.html

```diff
@@ -17,7 +17,9 @@
 
         // Wait for the SDK to be initialized
         VSS.ready(function () {
-            VSS.require(["../dist/app"], function (app) { });
+            VSS.require(["../dist/app"], function (app) { 
+                 VSS.notifyLoadSucceeded();
```

Comment:
Were you getting timeout errors from VSTS because this content was taking awhile to load?